### DISCUSSION TITLE
Table showing "encrypted" data with dbeaver.

### DISCUSSION THREAD

**Discussion by Eletropontorn** [dbeaver/dbeaver#23061] [General]
Hi, I've just started using dbeaver and i'm trying to manage a firebird database but i've got this problem. The dbeaver is showing the data like it's "encrypted".  The eg2 is another program, where i can see the data without problems. I'd appreciate any response. Thank you1 
![eg1](https://github.com/dbeaver/dbeaver/assets/162372558/2dbe714d-fcab-4d95-adac-ae9d0fad48d8)
![eg2](https://github.com/dbeaver/dbeaver/assets/162372558/a00288e0-590b-4865-98bb-f84502925e8a)

**Comment by LonwoLonwo** [comment]
Hello @abouliac 

Please check your data viewer format settings. 
That your string values are still strings (default) and not binaries.

![2024-03-06 10_03_09-Window](https://github.com/dbeaver/dbeaver/assets/45152336/ffc5a89f-72ab-4eaa-b05f-b673506940a0)

Also, you can change the data value to the database's native format.

![2024-03-06 09_59_28-Window](https://github.com/dbeaver/dbeaver/assets/45152336/82f78b11-4baa-4c6a-a952-77a8ed537820)

**Comment by Eletropontorn** [comment]
Hi, i appreciate your reply. Well, these are my settings. I've tried changing them from default to database native (display format) but it still not showing how it is supposed to. In the menu "set name transform" i've tried binary (all options, hex, native, strings)... But it still not showing. Then i returned it to default. I've no problems with Microsoft SQL Server's database. It happens only with this firebird database.

![eg2](https://github.com/dbeaver/dbeaver/assets/162372558/7f9a665d-242e-4077-8469-046b654806c7)
![eg1](https://github.com/dbeaver/dbeaver/assets/162372558/692f5597-9720-4bc1-b31d-18fac8983524)

**Reply by mkgrgis** [reply]
@abouliac , what is Firebird data type and subtype for the problem data column? 
Please refer Firebird [documentation](https://www.firebirdsql.org/file/documentation/chunk/en/refdocs/fblangref30/fblangref30-datatypes-bnrytypes.html) and try to read necessary description from  [query to metadata from SO answer](https://stackoverflow.com/questions/12070162/how-can-i-get-the-table-description-fields-and-types-from-firebird-with-dbexpr).
